thoth/storages/graph/postgres.py

```diff
@@ -6738,6 +6740,108 @@ def get_bloat_data(self) -> List[Dict[str, Any]]:
 
         return self._process_bloat_data_results(tables=tables)
 
+    def purge_solver_documents(
+        self, *, os_name: Optional[str] = None, os_version: Optional[str] = None, python_version: Optional[str] = None
+    ) -> int:
+        """Store and purge to be deleted solver documents to Ceph"""
+        self.connect()
```

Comment:
The convention is that a user of this adapter makes sure the adapter is connected:

```suggestion
```

Here and other methods as well.

Reply:
Thanks, I removed it!